### DISCUSSION TITLE
Image Modal

### DISCUSSION THREAD

**Discussion by rootisareservedword** [EvotecIT/PSWriteHTML#367] [Q&A]
Hi,

Is there a cmdlet that will create an image modal?

**Comment by rootisareservedword** [comment]
I think Add-HTMLScript is what I am looking for.

**Reply by PrzemyslawKlys** [reply]
What would be the use case for image modal? There's Winbox functionality but just not sure what you need.

**Reply by rootisareservedword** [reply]
I basically just want to insert a scaled down image of a screenshot and then enlarge it when it is clicked.

**Reply by PrzemyslawKlys** [reply]
I see, that's not implemented. Only custom solution. I was thinking on adding some kind of image gallery.

Something like: 
- https://codepen.io/seanfree/pen/BLrNzx
- http://miromannino.github.io/Justified-Gallery/
- https://fotorama.io/#3ed25902-4a51-4628-a057-1e55fbca7856
- http://brutaldesign.github.io/swipebox/
- https://blueimp.github.io/Gallery/

Thinking of it now you could probably use Kineto Carousel to show images.
